### DISCUSSION TITLE
cannot add Localhost

### DISCUSSION THREAD

**Discussion by fadai88** [Cyfrin/foundry-full-course-cu#1322] [General]
I cannot add localhost to my MM. I'm getting "Could not fetch chain ID. Is your RPC URL correct?" when I enter "31337" into the Chain ID field.
![Untitled](https://github.com/Cyfrin/foundry-full-course-f23/assets/12747028/6ba87931-1600-43d6-892e-6fe333ab6c00)

**Comment by ghost** [comment]
Usually this problem occurs when the local node is not running. Run `anvil` on the terminal and try again.

**Comment by GuiFlam** [comment]
I'm having the same problem. Have you ever found a solution?
